app/models/guide.rb

```diff
@@ -9,6 +9,8 @@ class Guide < ActiveRecord::Base
   has_many :editions, dependent: :destroy
   has_many :topic_section_guides, autosave: true
 
+  has_one :topic, through: :topic_section_guides
```

Comment:
There is an instance method, `#topic`, which will clash with this. Did we need some of the association-y properties?